packages/builder/src/steps/import.ts

```diff
@@ -33,8 +33,18 @@ export default {
   async getState(runtime: ChainBuilderRuntime, ctx: ChainBuilderContextWithHelpers, config: Config) {
     const cfg = this.configInject(ctx, config);
 
-    const preset = config.preset ?? 'main';
```

Comment:
This conditional wasnt necessary as this was already being done in the configInject, so long as the cfg variable was used then it would default to 'main' if the defined preset was invalid or nonexistent